### ISSUE TITLE
mobilefacenet vs insightface mobile

### ISSUE THREAD

**Issue by MyraBaba** [MirrorYuChen/mnn_example#28]
Hi,

mxnet insightface look better accuracy than the mobilefacenet for recognition..

do you think so ? if not what am I missing ?

Best

PS: can we use insightface mobile model with mnn ?

**Comment by MyraBaba** [comment]
@xuguozhi 

can we use insightface 512 feature model in MNN example.?

I tried but its only generate 128 feat

Best

**Comment by xuguozhi** [comment]
> @xuguozhi
> 
> can we use insightface 512 feature model in MNN example.?
> 
> I tried but its only generate 128 feat
> 
> Best

of course, you can try insightface 512 feature. But the ResNet50 is too large compared to mobilefacenet.

**Comment by MyraBaba** [comment]
I am new to the mobilefacenet . I indexed 15000 photos . Tested . Gives false positives . What will be the threshold for good similarity ? 1 is the %100 same pic it goes below even negatives so getting far away. 

One more question : the quality of the deleted face is also crucial for recognition quality. Ultraface detect many bully and bad faces. Is there any way to measure face quality (blurry, frontlines etc) . 

Examples:
